packages/splunkProcessor/src/splunkProcessor.js

```diff
@@ -132,7 +132,7 @@ function transformStepFunctionsLogEvent(logEvent) {
     eventMessage["X-Amzn-Trace-Id"] = normalizedHeaders["x-amzn-trace-id"]
     eventMessage["x-correlation-id"] = normalizedHeaders["x-correlation-id"]
     eventMessage["x-request-id"] = normalizedHeaders["x-request-id"]
-  } catch (_) {
```

Comment:
Is it me, or is this weirdly indented? Does this repo have an automatic linter? Also, if you'll allow me to nitpick a little, I'd space the try/catch and the triple-if on line 144 by one line, to distinguish them visually (extremely minor point)

Reply:
its weirdly formatted. Have corrected and pushed a change to enable format on save on this folder